tests/JiraClientStub.ts

```diff
@@ -56,6 +65,9 @@ export const jiraClientStub = {
     fields = fields ? JSON.stringify(fields) : 'null';
     console.log(`Invoked jira.moveIssue('${issueId}', '${transitionName}', ${fields})`);
   },
+  async transitionIssue(issueId: string, transition: any, fields: any = null): Promise<void> {
+    console.log(`${issueId}: Executing '${transition.name}' (${transition.id}) transition`);
```

Comment:
Use the same logging as every other `console.log`, to have the same format of `Invoked ... method name (serialized params)`